AutoROM/AutoROM.py

```diff
@@ -1,175 +1,188 @@
+                continue
+
+            # Read file from archive
+            fp = tarfp.extractfile(member)
+            bytes = fp.read()
+
+            # Get hash
+            md5 = hashlib.md5()
+            md5.update(bytes)
+            hash = md5.hexdigest()
+
+            if hash not in checksum_map:
+                warnings.warn(f"File {member.name} not supported.")
+                continue
+
+            # Get filename from checksum map
+            file_name = checksum_map[hash]
+
+            # Write ROM to all output folders
+            for package in packages:
+                rom_path = package.path / file_name
+                with rom_path.open("wb") as romfp:
+                    romfp.write(bytes)
+                if not package.filter(str(rom_path)):
+                    rom_path.unlink()
+                    continue
+
+                print(f"Installed {rom_path}")
+
+            # Cross off this ROM
+            del checksum_map[hash]
+
+
+SupportedPackage = namedtuple("SupportedPackage", ["path", "filter"])
+
+
+def find_supported_packages():
+    installation_dirs = []
+
+    # Try and find ale-py
     try:
-        import multi_agent_ale_py
-    except ImportError:
-        multi_ale_installed = False
 
-    installation_dirs = []
+        # isSupportedROM filter. There's some multi-agent games that aren't supported in ale-py.
+        def _ale_py_filter(path):
+            from ale_py import ALEInterface
 
-    if ale_installed:
-        ale_install_dir = ale_py.__file__
-        if ale_install_dir is not None:
-            ale_install_dir = ale_install_dir[:-11] + "ROM/"
-            installation_dirs.append(ale_install_dir)
-        else:
-            ale_installed = False
-    else:
-        ale_install_dir = None
-
-    if multi_ale_installed:
-        mulit_ale_install_dir = multi_agent_ale_py.__file__
-        if mulit_ale_install_dir  is not None:
-            mulit_ale_install_dir = mulit_ale_install_dir[:-11] + "ROM/"
-            installation_dirs.append(mulit_ale_install_dir)
-        else:
-            multi_ale_installed = False
+            return ALEInterface.isSupportedROM(path) is not None
+
+        installation_dirs.append(
+            SupportedPackage(resources.files("ale_py") / "roms", _ale_py_filter)
+        )
+    except ModuleNotFoundError:
+        pass
+
+    # Try and find multi-agent-ale-py
+    try:
+        # Assume all ROMs are supported
+        installation_dirs.append(
+            SupportedPackage(
+                resources.files("multi_agent_ale_py") / "roms", lambda _: True
+            )
+        )
+    except ModuleNotFoundError:
+        pass
+
+    return installation_dirs
+
+
+@click.command()
+@click.option(
+    "--accept-license",
+    is_flag=True,
+    default=False,
+    type=bool,
+    help="Accept license agreement",
+)
+@click.option(
+    "--install-dir",
+    default=None,
+    type=click.Path(exists=True),
```

Comment:
Is there a reason why there is no short version of these command line arguments?

Reply:
The current arguments, -d and -h are a bit unhelpful for sure, especially in scripts where you want to read the command as code (which is the main use of AutoROM right now anyways).